packages/menu/src/MenuItem.tsx

```diff
@@ -184,6 +189,7 @@ const MenuItem = React.forwardRef(
       disabled = false,
       active = false,
       className,
+      titleTextClassName,
```

Comment:
Before this is complete, I need to examine this, because for some reason in some cases this is showing up as an attribute `titletextclassname` on the link/button, rather than as part of the class composition by Emotion.

Reply:
Oh weird. I don't see anything particularly wrong here with that 🤷‍♂

Reply:
This will be addressed as part of removing MenuItem as a dependency for SideNavItem, so no worries